src/Microsoft.AspNetCore.HttpOverrides/project.json

```diff
@@ -24,10 +24,11 @@
   "dependencies": {
     "Microsoft.AspNetCore.Http.Extensions": "1.1.0-*",
     "Microsoft.Extensions.Logging.Abstractions": "1.1.0-*",
-    "Microsoft.Extensions.Options": "1.1.0-*"
+    "Microsoft.Extensions.Options": "1.1.0-*",
+    "System.Collections.Immutable": "1.2.0"
```

Comment:
Make it a readonly list. We're not going to take a dependency to expose a not so common exchange type.